autoleagueplay/replays.py

```diff
@@ -2,24 +2,71 @@
 from enum import Enum
 from pathlib import Path
 from typing import Dict, Any
+import os
+import RattletrapPython.rattletrap as rat
+import json
 
 import requests
 from rlbottraining.history.metric import Metric
+from autoleagueplay.paths import WorkingDir
 from watchdog.events import LoggingEventHandler
 from watchdog.observers import Observer
 
-
 class ReplayPreference(Enum):
     SAVE = 'save'  # save to the default replays directory
     CALCULATED_GG = 'calculated_gg'  # save locally and upload to https://calculated.gg/
     IGNORE_REPLAY = 'ignore'
+    ANONYMIZE_REPLAY = 'anonym'
 
 
 def upload_to_calculated_gg(replay_path: Path):
     with open(replay_path, 'rb') as f:
         response = requests.post('https://calculated.gg/api/upload', files={'replays': f})
         print(f'upload response to {replay_path.name}: {response}')
 
+def anonymize_replay(replay_path: Path, working_dir: WorkingDir):
+    replay_name = str(os.path.basename(str(replay_path)).split('.')[0])
```

Comment:
You already have the Path object, so you can just do
```suggestion
    replay_name = replay_path.stem
```